api/src/main/java/org/openmrs/module/locationbasedaccess/LocationBasedAccessActivator.java

```diff
@@ -49,13 +54,23 @@ public void willStart() {
      */
     public void started() {
 		LocationUtils.createPersonAttributeTypeForLocation();
+		generateOpenMRSPrivilegeForLocations();
 	    createGlobalPropertyForLocationAttribute(LocationBasedAccessConstants.LOCATION_ATTRIBUTE_GLOBAL_PROPERTY_NAME,
 			    LocationBasedAccessConstants.PERSONATTRIBUTETYPE_UUID);
 	    createGlobalPropertyForLocationAttribute(LocationBasedAccessConstants.REF_APP_LOCATION_USER_PROPERTY_NAME,
 			    LocationBasedAccessConstants.LOCATION_USER_PROPERTY_NAME);
 	    log.info("Location Based Access Control Module started");
     }
 
+	private void generateOpenMRSPrivilegeForLocations() {
+		UserService userService = Context.getUserService();
+		List<Location> openMrsLocations = Context.getLocationService().getAllLocations();
+		for(Location location : openMrsLocations){
```

Comment:
format this line, 
further is there any possibilities for `openMrsLocations` null?

Reply:
I don't think there will a point where OpenMRS with no Location